rq/job.py

```diff
@@ -158,6 +158,13 @@ def set_status(self, status, pipeline=None):
         connection = pipeline if pipeline is not None else self.connection
         connection.hset(self.key, 'status', self._status)
 
+    def get_meta(self, refresh=True):
```

Comment:
Can we test this function? You added a test case but I don't think `job.get_meta` is tested.

Reply:
Awkward, you're right. I fixed it.